components/pkg-export-kubernetes/src/manifestjson.rs

```diff
@@ -59,8 +59,8 @@ impl ManifestJson {
         for bind in &manifest.binds {
             let json = json!({
                 "name": bind.name.clone(),
-                "service": bind.service.clone(),
-                "group": bind.group.clone(),
+                "service": bind.service_group.service().to_owned(),
+                "group": bind.service_group.group().to_owned(),
```

Comment:
Does the bind need to be broken out into its constituent parts, or can it just be added to the JSON using its own string rendering?

There are other parts of a bind; this implementation is going to omit them entirely, and will also need to be updated when / if the parts of a bind change in the future. Admittedly, these other parts of a bind are not 100% fleshed out at the moment, but they are there.

Reply:
> Does the bind need to be broken out into its constituent parts, or can it just be added to the JSON using its own string rendering?

The hHbitat k8s operator keeps them divided in their API so unless that changes, we have to have them separated in here too.

> There are other parts of a bind; this implementation is going to omit them entirely, and will also need to be updated when / if the parts of a bind change in the future. Admittedly, these other parts of a bind are not 100% fleshed out at the moment, but they are there.

That will also have to change in the habitat (k8s) operator side.